### DISCUSSION TITLE
Pulseeffects sink-input Disappears

### DISCUSSION THREAD

**Discussion by jtharkey** [wwmm/easyeffects#1499] [General]
I'm running pulseeffects still on ubuntu 21.10, the version in the repo, and for some reason after about 5 seconds of running the Pulseeffects_apps sink-input just disappears. Everything else keeps running, it's not like pulseeffects dies. I cannot find the reason for this. Is there a log I can check? I haven't found one yet.

**Comment by wwmm** [comment]
Take a look at the `Basic Logs` section of our wiki https://github.com/wwmm/easyeffects/wiki/Reporting-Bugs and replace `easyeffects` by `pulseeffects` when running the commands to get the logs.

If `Pulseeffects_apps` is disappearing either PulseEffects is dying or something is unloading its virtual devices.

**Reply by jtharkey** [reply]
Will do, thank you.

**Reply by jtharkey** [reply]
Thank you again, that helped immensely. What seems to be happening is my sink-input (my app) starts and uses PulseEffects_app as a sink, but when I play my first sound it automatically switches sinks to the sound card, bypassing PulseEffects. Then PulseEffects_app goes away because nobody is using it. I'm confused about why that would happen. Is it because of rescue stream feature? I have that turned off in the conf file.
